src/com/rct/homework/lesson2/task08/Validator.java

```diff
@@ -0,0 +1,62 @@
+package com.rct.homework.lesson2.task08;
+
+import com.rct.homework.lesson2.task08.exceptions.ProductException;
+import com.rct.homework.lesson2.task08.exceptions.StorageException;
+
+import java.math.BigDecimal;
+
+/**
+ * Class for validation methods.
+ *
+ * @author Oleg Baslak
+ * @version 1.0
+ * @since 08.10.2016
+ */
+abstract public class Validator {
+
+    /**
+     * Checks if string that must be parsed to <code>BigDecimal</code>
+     * price can be parsed and if it can checks if price not <= 0.
+     *
+     * @param price price to check
+     * @return true - if price is valid, false - otherwise
+     * @throws ProductException if occurred parsing error or
+     *                          price <= 0
+     */
+    public static boolean isValidPrice(String price) throws ProductException {
```

Comment:
What if in future you will need to support products with positive price and... negative price (to show the debt for example). Is your implementation easily extendable? I would approach we discussed in skype (check < 0 in product specific constructor will be quite more extendable)